test/test-lib-mysql.sh

```diff
@@ -114,9 +114,9 @@ function test_mysql_s2i() {
 
 function test_mariadb_integration() {
   local service_name=mariadb
-  ct_os_template_exists mariadb-ephemeral && t=mariadb-ephemeral || t=mariadb-persistent
+  ct_os_template_exists mariadb-ephemeral && t=mariadb-ephemeral-template.json || t=mariadb-persistent-template.json
```

Comment:
Does it make sense to split tests are run as ephemeral as persistent?

Reply:
If I understand this correctly; if we decided to use local template `mariadb-ephemeral-template.json` we do not need to check with `ct_os_template_exists` if it already exist in the openshift do we? Then I would suppose using testing both templates.

However, haven't this test aimed at testing a template, which is already available in the openshift?

Reply:
Let's test both of them.

Reply:
And could you please explain to me, what is the difference between using the local template and the template available in the OCP?

Reply:
Thank you, I think I understand. In that case why do we not pull the templates from the repository but use the local ones?

Reply:
I expect we use local ones so we can test out changes to templates maintained in this repository as part of the PR.

Reply:
Makes sense, thanks.